apps/builder/components/configuration-sections/section-wallet-features.tsx

```diff
@@ -0,0 +1,76 @@
+import { UniqueIdentifier } from '@dnd-kit/core'
+import { useAppKitContext } from '@/hooks/use-appkit'
+import { WalletFeatureName } from '@/lib/types'
+import { ConstantsUtil, WalletFeature } from '@reown/appkit-core'
+import dynamic from 'next/dynamic'
+import { urlStateUtils } from '@/lib/url-state'
+
+const SortableWalletFeatureList = dynamic(
+  () =>
+    import('@/components/sortable-list-wallet-features').then(mod => mod.SortableWalletFeatureList),
+  { ssr: false }
+)
+
+const defaultWalletFeaturesOrder = ['onramp', 'swaps', 'receive', 'send']
+
+export function SectionWalletFeatures() {
+  const { config, updateFeatures } = useAppKitContext()
+  const connectMethodsOrder = config.features.walletFeaturesOrder || defaultWalletFeaturesOrder
+
+  function handleNewOrder(items: UniqueIdentifier[]) {
+    const titleValueMap = {
+      Buy: 'onramp',
+      Swap: 'swaps',
+      Receive: 'receive',
```

Comment:
Does `Receive` feature changes anything in the Modal? I could not see differences in the preview

Reply:
It's only visible on the embedded wallet connection. The default (old) account view doesn't have this as one of the options in the list